scripts/fbi/hate_crime/geo_id_resolver.py

```diff
@@ -43,14 +37,111 @@
 _IGNORE_CITIES = ['abington township pa',
                   'lone tree co']  # ignoring cities which have duplicates
 
-city = {}
-with open(os.path.join(city_geocodes_csv_path), encoding='utf8') as csvfile:
+# A mapping from city name to the dcid.
```

Comment:
Could you add a comment here on how this list is different from the csv? I'm a little confused about that myself.. I see that you made the change with [this commit](https://github.com/datacommonsorg/data/pull/624/commits/6bebfb8e59caf45374438c03aecea43ea4a82eb3) but i wonder if there's a simpler solution for this that's easy to maintain.

Reply:
The issue arises due to this [line](https://github.com/datacommonsorg/data/blob/master/scripts/fbi/crime/geocode_cities.py#L39) of code in geocode_cities.py

Some of the mappings in manual_geocodes.csv do not match with the ones we curated manually. Due to this, the above assertion fails and this causes the tests in geocodes_cities_test.py to fail.

One example would be `big bear ca` which maps to [geoId/0606406](https://datacommons.org/browser/geoId/0606406) in the hate crime import but manual_geocodes maps it to [geoId/606434](https://datacommons.org/browser/geoId/0606434).